src/adapter/src/catalog/storage.rs

```diff
@@ -1601,8 +1609,8 @@ impl_codec!(ItemValue);
 
 #[derive(Clone, Message, PartialOrd, PartialEq, Eq, Ord, Hash)]
 struct RoleKey {
-    #[prost(uint64)]
-    id: u64,
+    #[prost(string)]
```

Comment:
Oooh our first protobuf migration. The code here will cause any existing stash to fail on startup. We need to decide if we're ok with breaking the world again or not. I don't know.

Reply:
As per slack discussion, next release will have another breaking change, so we're going to piggy back off of that.